### PR TITLE
Add service rules for input table

### PR DESCRIPTION
**Pull request by rozdy**

### CODE REVIEW

File: src-java/rule-manager/rule-manager-api/src/main/java/org/openkilda/rulemanager/action/noviflow/PopVxlanAction.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import org.openkilda.rulemanager.action.ActionType;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize
+public class PopVxlanAction implements Action {
+
+    @Override
+    public ActionType getType() {
+        return ActionType.NOVI_POP_VXLAN;
```
[the remark this answers is not in the document]
Comment (rozdy) [reply src-java/rule-manager/rule-manager-api/src/main/java/org/openkilda/rulemanager/action/noviflow/PopVxlanAction.java]: Fixed.

File: src-java/rule-manager/rule-manager-api/src/main/java/org/openkilda/rulemanager/action/noviflow/Oxm.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.rulemanager.action.noviflow;
+
+public enum Oxm {
```
Comment (nikitamarchenko) [inline src-java/rule-manager/rule-manager-api/src/main/java/org/openkilda/rulemanager/action/noviflow/Oxm.java:18]: maybe you can find a better name for that class

File: src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/factory/generator/service/RoundTripLatencyRuleGenerator.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+import lombok.Builder;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+@Builder
+public class RoundTripLatencyRuleGenerator implements RuleGenerator {
```
Comment (nikitamarchenko) [inline src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/factory/generator/service/RoundTripLatencyRuleGenerator.java:55]: I propose to move all the features based on noviflow into a separate package.

**Review by nikitamarchenko — APPROVED** [review]

**Review by dpoltavets — APPROVED** [review]

File: src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/factory/generator/service/RoundTripLatencyRuleGenerator.java (hunk 3 above)
Comment (rozdy) [reply src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/factory/generator/service/RoundTripLatencyRuleGenerator.java]: Moved.

**Pull request merged**